src/test/java/previewcode/backend/services/DatabaseServiceTest.java

```diff
@@ -153,6 +152,36 @@ public void insertsNewGroupsWithHunks() throws Exception {
     }
 
     @Test
+    public void insertsDefaultGroup() throws Exception {
+        PullRequestGroup group = new PullRequestGroup(new GroupID(42L), "Group A", "Description A", true);
+        List<OrderingGroup> defaultGroup = List.of(new OrderingGroupWithID(group, hunkIDs.map(id -> id.checksum).toJavaList()));
+        Action<Unit> dbAction = service.insertDefaultGroup(pullIdentifier, defaultGroup);
+
+        Collection<PullRequestGroup> groupsAdded = Lists.newArrayList();
+
+        Interpreter interpreter =
+                interpret()
+                        .on(InsertPullIfNotExists.class).returnA(pullRequestID)
+                        .on(FetchGroupsForPull.class).returnA(List.empty())
```

Comment:
If you remove this line, the test will fail if the insertion of a default group depends on existing groups. This would catch the unnecessary `clearExistingGroups` action in DatabaseService.